cmd/config/set/config_set.go

```diff
@@ -0,0 +1,44 @@
+package set
+
+import (
+	"errors"
+
+	"numerous.com/cli/cmd/output"
+	"numerous.com/cli/internal/config"
+)
+
+var (
+	ErrInvalidPropertyToSet = errors.New("invalid property to set")
+	ErrInvalidArgumentCount = errors.New("expected exactly two arguments")
+)
+
+func configSet(args []string) error {
+	if len(args) != 2 { // nolint:mnd
+		output.PrintError("Expected exactly two arguments, got %d", "", len(args))
+		return ErrInvalidArgumentCount
+	}
+	property, value := args[0], args[1]
+
+	cfg := config.Config{}
+	if err := cfg.Load(); err != nil {
+		output.PrintErrorDetails("Error loading configuration", err)
+		return err
+	}
+
+	switch property {
+	case "organization":
+		cfg.OrganizationSlug = value
+	default:
+		output.PrintError("Invalid property to set: %q", "", property)
+		return ErrInvalidPropertyToSet
+	}
+
+	if err := cfg.Save(); err != nil {
+		output.PrintErrorDetails("Error saving configuration", err)
+		return err
+	}
+
+	output.PrintlnOK("Set %s=%s", property, value)
```

Comment:
consider this, just so it's easier to see if you accidently it pasted in some specialt whitespace character?
```suggestion
	output.PrintlnOK("Set %s=%q", property, value)
```

Reply:
Sure. We do need to consider if this is appropriate if any non-string values are added later :)